systemtests/tests/py2plug-fd-local-fileset-basic/CMakeLists.txt

```diff
@@ -17,24 +17,26 @@
 #   Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
 #   02110-1301, USA.
 
-if(TARGET python-fd)
-  create_systemtest(${SYSTEMTEST_PREFIX} "py2plug-fd-local-fileset-basic")
+get_filename_component(BASENAME ${CMAKE_CURRENT_BINARY_DIR} NAME)
+
+if(TARGET python-fd AND BASENAME MATCHES "py2")
```

Comment:
That doesn't have the intended effect, because it will add a disabled python-test when run in the py3 directory.
```
if(BASENAME MATCHES "py2")
  if(TARGET python-fd)
    create_systemtest(...)
  else()
    create_systemtest(... DISABLED)
  endif()
endif()
```
Same applies for py3 below.

Reply:
Makes sense, done